data/lib/api/team/team_model.dart

```diff
@@ -18,31 +20,38 @@ class TeamModel with _$TeamModel {
     String? profile_img_url,
     String? created_by,
     DateTime? created_at,
-    List<UserModel>? players,
+    @JsonKey(name: FireStoreConst.teamPlayers)
```

Comment:
what does it do?

Reply:
It retrieves the value based on the Firebase key name, so we don't have to update the players field name in all places.